lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml

```diff
@@ -6,16 +6,17 @@ _template:
   get_command: "show running-config interface all | section 'interface nve'"
   get_context:
     - '/^interface <name>$/i'
-    - '/^member vni <vni> ?(associate-vrf)?$/'
+    - '(?)/^member vni <vni> ?(associate-vrf)?$/'
```

Comment:
It makes the line optional. The test was failing during the `all_vnis` call because it doesn't include a vni since it's trying to collect all vnis.